CONTRIBUTE.md

```diff
@@ -0,0 +1,45 @@
+# Darts Contribution Guidelines
+
+## Main Development Principles
+
+* Focus on simplicity and clarity for end users.
+* Pay attention to designing the best possible API, simple by default, but offering control when needed.
+* Make it hard for users to make mistakes (e.g., future data leakage).
+* These principles should prevail even if
```

Comment:
unless you meant this to apply only to the first rules, I'd move it to the end of the list and make outstanding, eg. bolded, quoted, etc.